kaolin/datasets/modelnet.py

```diff
@@ -12,28 +12,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import Callable, Iterable, Optional, Union
+from typing import Callable, Iterable, Optional, Union, List
 
 import torch
 import os
-import sys
 from glob import glob
-import scipy.io as sio
+from tqdm import tqdm
 
-import kaolin as kal
+from kaolin.rep.TriangleMesh import TriangleMesh
+from kaolin.transforms import transforms as tfs
 
 
-_MODELNET10_CLASSES = ['bathtub', 'bed', 'chair', 'desk', 'dresser', 'monitor',
-                       'night_stand', 'sofa', 'table', 'toilet']
-
-
-class ModelNet10(torch.utils.data.Dataset):
-    r"""Dataset class for the ModelNet10 dataset.
+class ModelNet(object):
```

Comment:
I believe ModelNet is now compatible with both ModelNet10 and ModelNet40. But ligne 30 is still `basedir (str): Path to the base directory of the ModelNet10 dataset.`

Reply:
Agreed, fixed.